R/neighborhood_analysis.R

```diff
@@ -274,6 +274,10 @@ RegionNeighbors.Seurat <- function (
 
   # Add data to Seurat object
   nbs_rearranged <- do.call(bind_cols, lapply(column_labels, function(lbl) {
+    if (grepl("\\s", lbl)) {
+      new_lbl <- gsub(" ", "\\.", lbl)
+      cli_alert_warning("Blankspaces detected among column labels. New column name set to '{new_lbl}'.")
+    }
```

Comment:
Perhaps it's better to be strict and abort is there are blankspaces detected. I think it's bad practice to use those but you should decide.

Reply:
Now I changed it to what you suggested instead to perform a check and abort if any blank spaces are found. I'll push it soon